server/templates/base.html

```diff
@@ -73,6 +73,21 @@
     globalThis.isCustomDC = {{ config['CUSTOM']|int }};
     globalThis.STAT_VAR_HIERARCHY_CONFIG = {{ config['STAT_VAR_HIERARCHY_CONFIG'] | tojson }};
   </script>
+  <script>
+    // Post the iframe's URL to the parent
+    window.addEventListener('message', function(event) {
+      // MUST verify the origin to be autopush DC
+      if (event.origin === 'https://autopush.datacommons.org') {
```

Comment:
What's the check for autopush.datacommons.org for?

Reply:
basically only allow autopush to host this comparator app